src/dodal/devices/apple2_undulator.py

```diff
@@ -373,16 +384,20 @@ def __init__(
                 float, initial_value=None
             )
 
-        with self.add_children_as_readables():
-            # Store the polarisation for readback.
-            self.polarisation, self._polarisation_set = soft_signal_r_and_setter(
-                str, initial_value=None
-            )
+        # Store the polarisation for setpoint.
+        self.polarisation_setpoint, self._polarisation_setpoint_set = (
+            soft_signal_r_and_setter(Pol)
+        )
```

Comment:
Should: You can just use the setter added in https://github.com/DiamondLightSource/dodal/pull/1078 for this, right?